responses/__init__.py

```diff
@@ -252,11 +258,11 @@ def __len__(self) -> int:
         return len(self._calls)
 
     @overload
-    def __getitem__(self, idx: int) -> Call:
+    def __getitem__(self, idx: int) -> Call:  # type: ignore[misc]
```

Comment:
too many ignores, something looks to be incorrect

Reply:
fixed in https://github.com/getsentry/responses/pull/684

first merge https://github.com/getsentry/responses/pull/684, then current PR and we are good to ship the release

Reply:
Merged, thanks!